docs/en/observability/monitor-uptime-synthetics.asciidoc

```diff
@@ -1,50 +1,121 @@
+=== Lightweight HTTP/S, TCP, and ICMP checks
+
+In the {uptime-app}, you can monitor the status of network endpoints using the following lightweight checks:
+
+|===
+
+| *ICMP monitor* | Check the availability of your hosts. The ICMP monitor uses ICMP (v4 and v6) Echo
+Requests to check the network reachability of the hosts you are pinging. This will tell you whether the
+host is available and connected to the network. However, it doesn't tell you if a service on the host is running or
+not.
+
+| *TCP monitor* | Monitor the services running on your hosts. The TCP monitor checks individual ports
+to ensure the service is accessible and running.
+
+| *HTTP monitor* | Monitor your website. The HTTP monitor checks to ensure specific endpoints return the correct
+status code and display the correct text.
+
+|===
+
+[role="screenshot"]
+image::images/uptime-app.png[Uptime app in Kibana]
+
+
+// Along with getting notified when your <<tls-certificate-alert,TLS certificates>> are set to expire, you can also
+// create an <<duration-anomaly-alert,Uptime duration anomaly >> rule to receive notifications based on the response durations for all of the
+// geographic locations of each monitor.
+
+// , explore
+// endpoint status over time, drill down into specific monitors, and view a high-level
+// snapshot of your environment at any point in time.
 
 [discrete]
 [[monitoring-synthetics]]
-== Browser checks
+// [[synthetic-monitoring]]
+=== Browser checks
+
+// | *Browser monitor* | 
 
 beta[] Real browser synthetic monitoring enables you to test critical actions and requests that an end-user would perform
-on your site at predefined intervals and in a controlled environment. You can view each synthetic monitoring journey
-in the {uptime-app} side-by-side with your other <<monitor-uptime,Uptime monitors>>. The result is rich, consistent, and repeatable
-data that you can trend and alert on.
+on your site at predefined intervals and in a controlled environment.
+The result is rich, consistent, and repeatable data that you can trend and alert on.
 
-For example, test popular user journeys, like logging in, adding items to a cart, and checking
+For example, you can test popular user journeys, like logging in, adding items to a cart, and checking
 out -- actions that need to work for your users consistently.
 
-Alerting ensures that any degraded performance or broken actions are fixed before impacting your bottom line or customer
-experience.
+Using the synthetics agent, you can run automated synthetic monitoring test suites on a real Chromium browser and
+view each synthetic monitoring journey in the {uptime-app} side-by-side with your other monitors.
 
 [role="screenshot"]
 image::images/synthetic-app-overview.png[Synthetics app overview]
 
-For more details, refer to <<synthetic-monitoring,Real browser synthetic monitoring>>.
+Alerting ensures that any degraded performance or broken actions are fixed before impacting your bottom line or customer
+experience.
```

Comment:
Nit: to be precise here I'd say that "alerting _helps_ resolving", not that it resolves. I.e.

> Alerting helps you detect degraded performance or broken actions before your users do. By receiving alerts early, you can fix issues before they impact your bottom line or customer experience.